pkg/util/util.go

```diff
@@ -31,6 +32,25 @@ const (
 	blockdevFileName = "/usr/sbin/blockdev"
 )
 
+var (
+	findMntByPath = func(path string) ([]byte, error) {
+		return exec.Command("/usr/bin/findmnt", "-J", path).CombinedOutput()
+	}
+
+	getHwSectorSize = func(deviceName string) ([]byte, error) {
+		return exec.Command("cat", fmt.Sprintf("/sys/block/%s/queue/hw_sector_size", deviceName)).CombinedOutput()
+	}
+
+	getlsblk = func() ([]byte, error) {
+		return exec.Command("/usr/bin/lsblk", "-J").CombinedOutput()
```

Comment:
I am remembering #1258  (use blockdev instead of lsblk due to limitations with dev-mapper) and I wonder whether we should use blockdev here for the same reason.

Reply:
I am not using lsblk to get size, I am using lsblk to find all the block devices, so I can then use the name to find the actual block size.

Reply:
Also note we can't use blockdev, because it requires a device to be passed. But in certain scenarios (like lets say local storage as in our tests) the actual device is /dev/vda, but that device doesn't exist inside the container (no /dev/vda available to pass to blockdev). Fortunately /sys/block/<device> is available and contains the HW block size.